app/controllers/api/v1/accounts_controller.rb

```diff
@@ -15,6 +16,16 @@ def verify_credentials
     render action: :show
   end
 
+  def update_credentials
+    @account = current_user.account
+
+    if @account.update_attributes(account_params)
```

Comment:
Okay one last thing I noticed!

You can use `@account.update!(account_params)` here without the if/else construct. It will throw an `ActiveRecord::RecordInvalid` exception if the update fails, which is caught in `ApiController` to display a standardized error message.